borg/_hashindex.c

```diff
@@ -77,6 +79,7 @@ static int hash_sizes[] = {
 
 #define HASH_MIN_LOAD .25
 #define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_EFF_LOAD .85
```

Comment:
Mmh but he also mentions that due to lookups happening more often than ins/del (which is also true for us)...

> In real practice, except insertions and removals there are many lookups, making rehash cost relatively less meaningful, therefore optimal _effective loads to rehash at should be lower, than in the 3rd and 4th columns of the table above._